core/src/main/java/org/apache/calcite/adapter/java/ReflectiveSchema.java

```diff
@@ -320,6 +326,8 @@ private static class ReflectiveTable
    * &nbsp;
    *   Employee[] EMPS;
    *   Department[] DEPTS;
+   *   List&lt;Discount&gt; DISCOUNTS;
```

Comment:
was this an omission?

Reply:
Im really sorry for the long inactive time.

There is checkstyle rule and wo `&lt;` I have an error `Javadoc comment at column 11 has parse error. Missed HTML close tag 'Offer'. Sometimes it means that close tag missed for one of previous tags.`